--- CMakeLists.txt.orig	2025-10-29 14:08:29
+++ CMakeLists.txt	2025-10-29 14:12:52
@@ -235,12 +235,14 @@
 endif()
 
 eprosima_find_package(fastcdr 2 REQUIRED)
-eprosima_find_thirdparty(Asio asio VERSION 1.13.0)
+set(THIRDPARTY_Asio FORCE CACHE STRING "Force use of thirdparty Asio")
+include_directories(thirdparty/asio)
+set(Asio_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/thirdparty/asio/include")
 eprosima_find_thirdparty(TinyXML2 tinyxml2)
 
 find_package(foonathan_memory REQUIRED)
 message(STATUS "Found foonathan_memory: ${foonathan_memory_DIR}")
-find_package(ThirdpartyBoost REQUIRED)
+include_directories(thirdparty/boost/include)
 
 if(ANDROID)
     if((ANDROID_PLATFORM LESS_EQUAL 23) OR (ANDROID_NATIVE_API_LEVEL LESS_EQUAL 23))
