--- ./src/ros2/orocos_kdl_vendor/python_orocos_kdl_vendor/CMakeLists.txt
+++ ./src/ros2/orocos_kdl_vendor/python_orocos_kdl_vendor/CMakeLists.txt
@@ -45,11 +45,8 @@ macro(build_pykdl)
   # This version must match orocos_kdl_vendor exactly
   fetchcontent_declare(
     python_orocos_kdl
-    URL https://github.com/orocos/orocos_kinematics_dynamics/archive/ce4bcb65a050615b6d7f21bc60fbb2656515791b.zip
-    URL_MD5 f273799921a2fd21b79211c93fa67a5d
-    PATCH_COMMAND
-      ${CMAKE_COMMAND} -E chdir <SOURCE_DIR> git apply -p1 --ignore-space-change --whitespace=nowarn
-        ${CMAKE_CURRENT_SOURCE_DIR}/patches/0001-fix-windows-debug.patch
+    URL https://github.com/orocos/orocos_kinematics_dynamics/archive/502976efb0153b84591a343ea0a726b14ba0a2b2.zip
+    URL_MD5 992853096cef7fc995407b43c9ceb245
   )
 
   fetchcontent_getproperties(python_orocos_kdl)