--- install_venv/lib/python3.11/site-packages/setuptools/command/easy_install.py
+++ install_venv/lib/python3.11/site-packages/setuptools/command/easy_install.py
@@ -153,11 +153,11 @@
     create_index = PackageIndex
 
     def initialize_options(self):
-        warnings.warn(
-            "easy_install command is deprecated. "
-            "Use build and pip and other standards-based tools.",
-            EasyInstallDeprecationWarning,
-        )
+        # warnings.warn(
+        #     "easy_install command is deprecated. "
+        #     "Use build and pip and other standards-based tools.",
+        #     EasyInstallDeprecationWarning,
+        # )
 
         # the --user option seems to be an opt-in one,
         # so the default should be False.
