--- ./src/ros2/rviz/rviz_default_plugins/CMakeLists.txt
+++ ./src/ros2/rviz/rviz_default_plugins/CMakeLists.txt
@@ -242,6 +242,7 @@
 target_include_directories(rviz_default_plugins PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/${PROJECT_NAME}>
+  "/opt/homebrew/Cellar/orocos-kdl/1.5.3/include/"
 )
 
 target_link_libraries(rviz_default_plugins PUBLIC
